packages/react-server-test-pages/pages/navigation/playground.js

```diff
@@ -131,8 +105,6 @@ export default class NavigationPlaygroundPage {
 					<li>CT: Client Transition</li>
 					<li>RD: Reuse DOM</li>
 					<li>BD: Bundle Data</li>
-					<li>FB: Frameback</li>
-					<li><span className='not-available'>FB</span>: Frameback disabled (already in a frame)</li>
```

Comment:
There should be some styling (strikethrough, I think) associated with the `not-available` class that can be removed from the CSS, too.